src/sentry/api/urls.py

```diff
@@ -2256,6 +2259,11 @@
         ProjectPerformanceIssueSettingsEndpoint.as_view(),
         name="sentry-api-0-project-performance-issue-settings",
     ),
+    url(
+        r"^(?P<organization_slug>[^\/]+)/(?P<project_slug>[^\/]+)/performance-issues/configure/admin/$",
```

Comment:
The url ending with admin resource means that this endpoint is adding admin on put, while from the serializer I can see it's updating a bunch of configs. Can you explain what you're trying to do with this vs /configure?